script.js

```diff
@@ -0,0 +1,177 @@
+            const weatherDisplay = document.createElement('div')
+            weatherDisplay.className = 'weatherDisplay'
+            const cTempHRound = Math.round(dayTemp.maxtemp_c)
+            const cTempLRound = Math.round(dayTemp.mintemp_c)
+            const fTempHRound = Math.round(dayTemp.maxtemp_f)
+            const fTempLRound = Math.round(dayTemp.mintemp_f)
+            weatherDisplay.innerHTML = `
+                <h5>${locationDetails?.name || ''},  ${locationDetails?.country || ''}</h5>
+                <img src='${dayReport.day.condition.icon}'>
+                <h4>${dayReport.day.condition.text}</h4>
+                <h4>${dayReport.date}</h4>
+                <div class='c-temp'>
+                    <h3 class='temp-c'>H: ${cTempHRound}°c</h3>
+                    <h3 class='temp-c'>L: ${cTempLRound}°c</h3>
+                    </div>
+                    <div class='f-temp'>
+                    <h3 class='temp-f'>H: ${fTempHRound}°f</h3>
+                    <h3 class='temp-f'>L: ${fTempLRound}°f</h3>
+                </div>`
+            weatherResults.appendChild(weatherDisplay)
+        })     
+    }
+    toggleTemp()
+}
+
+function onError(error){
+    const container = document.querySelector('.container')
+    const errorDisplay = document.createElement('h3')
+    errorDisplay.className = 'error'
+    errorDisplay.innerText = error
+    errorDisplay.style.color = 'red'
+    container.appendChild(errorDisplay)
+}
+
+function errorDisplay(){
+    const errorDisplay = document.querySelector('.error')
+    if (errorDisplay){
+        errorDisplay.remove()
+    }
+}
+
+function clearSearch(){
+    form = document.querySelector('form')
+    const days = document.querySelector('.num-days')
+    form.reset()
+    days.style.display = 'none'
+}
+
+function resetDisplay(){
+    const weatherDisplayDiv = document.querySelector('.weatherDisplay')
+    if(weatherDisplayDiv){
+        weatherDisplayDiv.remove()
+    }
+}
+
+function toggleTemp(){
+    const cTemp = document.querySelectorAll('.c-temp')
+    const fTemp = document.querySelectorAll('.f-temp')
+    const currentTempC = document.querySelector('.current-temp-c')
+    const currentTempF = document.querySelector('.current-temp-f')
+    
+    cTemp.forEach((element) => {
+        element.addEventListener('click', () => {
+            element.style.display = 'none'
+            const correspondingFTemp = element.nextElementSibling
+            correspondingFTemp.style.display = 'flex'
+        })
+    })
+    
+    fTemp.forEach((element) => {
+        element.addEventListener('click', () => {
+            element.style.display = 'none'
+            const correspondingCTemp = element.previousElementSibling
+            correspondingCTemp.style.display = 'flex'
+        })
+    })
+
+    currentTempC.addEventListener('click', () =>{
+        currentTempC.style.display = 'none'
+        const correspondingTempF = currentTempC.nextElementSibling
```

Comment:
This variable should have been named correspondingTempC and not correspondingTempF, right?